functions/src/tests/index.test.ts

```diff
@@ -0,0 +1,87 @@
+import fetch from "node-fetch";
+import { API_BASE_URL, createUniqueId } from "../utils";
+import {
+  TEST_PAYLOAD,
+  TEST_SHARE_LINK_URL,
+  registerUrl,
+  INVALID_URL_ERROR,
+  URL_NOT_FOUND_ERROR,
+} from "./utils";
+
+describe("POST /registerUrl/", () => {
+  test("it returns a 200 and creates a new share link from valid params.", async () => {
```

Comment:
nit: I'm not 100% sure, but I think the describe / test names are meant to sort of read as a coherent sentence when combined.  So this would read as "POST /registerUrl/ it returns a 200 ..."  and so I would drop the "it."  But I could be out-of-touch with current JavaScript testing conventions.

Reply:
That makes sense--that flows better, so I've updated them

I was reading/formatting it like `test it returns....` I didn't follow any styleguide/strict convention